packages/block-library/src/navigation-link/edit.js

```diff
@@ -304,30 +296,43 @@ function NavigationLinkEdit( {
 	);
 }
 
+// function getRGB
```

Comment:
anything actionable with this comment or can it be removed?

Reply:
need to be removed 👍